<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanelWgt</class>
 <widget class="QWidget" name="ControlPanelWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="btnStartExperiment">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/play_exp.png</normaloff>:/icons/play_exp.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnContinueExperiment">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/skip_exp.png</normaloff>:/icons/skip_exp.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnPauseExperiment">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/pause_exp.png</normaloff>:/icons/pause_exp.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnStopExperiment">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/stop_exp.png</normaloff>:/icons/stop_exp.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>95</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
